apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -3,16 +3,18 @@ import _ from 'lodash';
 const SET_STUDIO_URL = 'teacherDashboard/SET_STUDIO_URL';
```

Comment:
Unrelated to this particular change, but I'm not sure what the right place for this file actually is. It's currently location doesnt seem quite right, as it's not a template, but having it colocated next to all the files it's providing state for does seem valuable. Thoughts?